### ISSUE TITLE
Flags rule PD011 but it is not correct

### ISSUE THREAD

**Issue by ghsanti** [astral-sh/ruff#13495]
* Rule PD011: https://docs.astral.sh/ruff/rules/pandas-use-of-dot-values/
* Code does not use pandas

```py
a = list()
a.values
```

> Use `.to_numpy()` instead of `.values`

* It will always happen in cases where `a` is a `class` that happens to have `values` property.

Maybe as a first fix it can check for the `pandas` module, I'm not using it, just as in the snippet above.

**Comment by MichaReiser** [comment]
Thanks. Yeah, that makes sense. We're working on type inference that will improve the precision of panda's rules. 

For now, I recommend you to disable the rule by using `ruff.lint.ignore = ["PD011"])`. 

I merge this issue into https://github.com/astral-sh/ruff/issues/6432 which discusses the problem holistically for all pandas rules.
